tests/infra/consortium.py

```diff
@@ -271,7 +271,7 @@ def get_member_by_service_id(self, service_id):
         )
 
     def vote_using_majority(
-        self, remote_node, proposal, ballot, wait_for_global_commit=True, timeout=3
+        self, remote_node, proposal, ballot, wait_for_commit=True, timeout=3
```

Comment:
There are dozens of references to "global_commit" in test code, if we're going to clean it up (which is fine), I would rather have it done in a separate PR, rather than have bits done here and there.